### PR TITLE
Fixed formatting shortcut and added shortcut for toggling comments

### LINKED ISSUE #129: Keyboard shortcut: comment line
It would be nice if we could use cmd + / to comment queries.

### PR DESCRIPTION
**Pull request by UtechtDustin**
This PR will add add a shortcut (`Ctrl+/`) and will Fix #129.
Also this PR will fix the shortcut for formatting code.

Peek 2020-08-17 23-13

### CODE REVIEW

**Comment by UtechtDustin** [comment]
@rathboma how i can trigger the checks to run again ?
There was a problem with the internet connection.

**Comment by rathboma** [comment]
I have to do it, don't worry that much though, usually I just ignore those types of failures

File: src/components/TabQueryEditor.vue [hunk 1]
```diff
@@ -105,13 +105,14 @@
   import _ from 'lodash'
   import 'codemirror/addon/search/searchcursor'
   import CodeMirror from 'codemirror'
+  require('codemirror/addon/comment/comment')
```
Comment (rathboma) [inline src/components/TabQueryEditor.vue:108]: `import 'codemirror/addon/comment/comment'`

File: src/components/TabQueryEditor.vue [hunk 2]
```diff
@@ -465,8 +475,10 @@
           "Cmd-Enter": this.submitTabQuery,
           "Ctrl-S": this.triggerSave,
           "Cmd-S": this.triggerSave,
-          "Ctrl+Shift+F": this.formatSql,
-          "Cmd+Shift+F": this.formatSql
+          "Shift-Ctrl-F": this.formatSql,
```
Comment (rathboma) [inline src/components/TabQueryEditor.vue:478]: doh, thanks

File: src/components/TabQueryEditor.vue (hunk 1 above)
Comment (UtechtDustin) [reply src/components/TabQueryEditor.vue]: Sorry, its the first project with `import/require`, i tried stuff like:
```
import Comment From '...'
```
Which dont worked but, so i thought require is the right way.
Thanks for the info :+1: 

Btw. i changed that.

**Pull request merged**